roofit/roostats/src/AsymptoticCalculator.cxx

```diff
@@ -1440,22 +1433,23 @@ RooAbsData * AsymptoticCalculator::MakeAsimovData(const ModelConfig & model, con
          std::unique_ptr<RooArgSet> cpars(cterm->getParameters(&gobs));
          std::unique_ptr<RooArgSet> cgobs(cterm->getObservables(&gobs));
          if (cgobs->getSize() > 1) {
-            oocoutE((TObject*)0,Generation) << "AsymptoticCalculator::MakeAsimovData: constraint term  " <<  cterm->GetName()
+            oocoutE(static_cast<TObject*>(0),Generation) << "AsymptoticCalculator::MakeAsimovData: constraint term  " <<  cterm->GetName()
```

Comment:
```suggestion
            oocoutE(static_cast<TObject*>(nullptr),Generation) << "AsymptoticCalculator::MakeAsimovData: constraint term  " <<  cterm->GetName()
```
If you are already changing a line where `0` is used instead of the C++ `nullptr`, please change it to `nullptr` in the same go.